OPHD/States/PlanetSelectState.cpp

```diff
@@ -90,9 +92,9 @@ PlanetSelectState::~PlanetSelectState()
 
 namespace {
 	std::vector<Planet::Attributes> planetAttributes = {
-		{ Planet::Attributes{Planet::PlanetType::Mercury, "planets/planet_d.png", 1, 10} },
-		{ Planet::Attributes{Planet::PlanetType::Mars, "planets/planet_c.png", 4, 30} },
-		{ Planet::Attributes{Planet::PlanetType::Ganymede, "planets/planet_e.png", 2, 15} }
+		{ Planet::Attributes{Planet::PlanetType::Mercury, "planets/planet_d.png", Planet::Hostility::High, 1, 10, "maps/merc_01", "tsets/mercury.png"} },
+		{ Planet::Attributes{Planet::PlanetType::Mars, "planets/planet_c.png", Planet::Hostility::Low, 4, 30, "maps/mars_04", "tsets/mars.png"} },
+		{ Planet::Attributes{Planet::PlanetType::Ganymede, "planets/planet_e.png", Planet::Hostility::Medium, 2, 15, "maps/ganymede_01", "tsets/ganymede.png"} }
```

Comment:
Intention is to move all this into an xml file eventually. It sounds like NAS2D will have some xml updates occurring so trying to wait for that before moving to xml file.